style.css

```diff
@@ -32,6 +31,11 @@
   height: 15vh;
   display: flex;
   align-items: center;
+  z-index: 2;
+  top: 0;
+  left: 0;
+  position: fixed;
+  background-color: var(--white-color);
```

Comment:
As there are multiple properties here you could sort them alphabetically which helps in debugging and code maintainability.

Reply:
thanks will keep in mind